### DISCUSSION TITLE
Notable differences between Sqflite and sqflite_common_ffi

### DISCUSSION THREAD

**Discussion by bunnyian** [tekartik/sqflite#1110] [Q&A] [answered]
Hello,

I am wondering if there are any important differences to be aware of between the two similar packages. At first glance the api looks the same and the only differences (from the readmes) is the additional steps to use on Linux/Windows.

Is this correct? Are there any changes to be made to migrate from sqflite to sqflite_common_ffi? Thanks.

**Comment by alextekartik** [comment, marked as answer]
The difference is in the implementation but the API is indeed similar (just a setup change).
The steps are described here: https://github.com/tekartik/sqflite/blob/master/sqflite_common_ffi/doc/using_ffi_instead_of_sqflite.md
